source/github-issue-link-status.js

```diff
@@ -144,13 +146,29 @@ function onNewComments(cb) {
 
 		// When you edit your own comment
 		commentList.addEventListener('submit', () => setTimeout(cb, 1000)); // Close enough
+
+		// When you click Preview in the editor
+		commentList.addEventListener('click', event => {
+			if (event.target.classList.contains('preview-tab')) {
+				setTimeout(cb, 1000); // Close enough
+			}
+		});
 	}
 }
 
+function onPreviewTab(cb) {
+	setTimeout(cb, 1000);
+}
+
 async function init() {
 	({token} = await options.getAll());
 	if (token) {
-		onAjaxedPages(() => onNewComments(apply));
+		onAjaxedPages(() => {
+			onNewComments(apply);
+			delegate('#issuecomment-new, .timeline-new-comment, .timeline-comment', 'button.preview-tab:not(.selected)', 'click', () => {
```

Comment:
This code is attaching a new event listener to `document` every time a new ajaxed page loads, and the event listeners pile up (which means one click will call the function multiple times)

To avoid this, we move the handler to the top level so `addEventListener` and `delegate` recognize they're the same exact function and ignore it. https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener